### PR TITLE
Feature/add india : Added Country India in CountriesList.kt with Popular Tourist Places

### PR DESCRIPTION
**Pull request by abhishektiwarijr**
1. Added India to the country list.
2. Added 8 tourist places with reference images.
3. Integration tested on Emulator (Android) and Web (Chrome).

Observations: 
1. On Web > Main Screen with Horizontal Cards > Horizontal Scrolling not working on drag but with Mouse wheel
2. Short Description Text on Main Cards are not clearly visible over the image.

### CODE REVIEW

**Comment by SEAbdulbasit** [comment]
Hi @abhishektiwarijr,
Thanks for the contribution and raising the issue. I also noticed that the text contract is not good. We will look into that.

File: gradlew.bat
Comment (SEAbdulbasit) [inline gradlew.bat:1]: Can we revert this file ?

**Review by SEAbdulbasit — CHANGES_REQUESTED** [review]
Also, as i merged another branch into master, it had some conflicts. I resolved, hopefully these are good.

**Review by SEAbdulbasit — APPROVED** [review]

**Comment by abhishektiwarijr** [comment]
Oh you resolved it. Thanks a lot @SEAbdulbasit. I love your work on KMP. Learning a lot from you.

**Comment by SEAbdulbasit** [comment]
Thank you @abhishektiwarijr, also the vertical scroll issue on Desktop is fixed.

**Comment by abhishektiwarijr** [comment]
> Thank you @abhishektiwarijr, also the vertical scroll issue on Desktop is fixed.

Awesome. Thanks

**Pull request merged**